.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "solidity-ibc-eureka"]
 	path = solidity-ibc-eureka
-	url = https://github.com/cosmos/solidity-ibc-eureka
+	url = https://github.com/rootulp/solidity-ibc-eureka
+	branch = rp/update-ibc-go
```

Comment:
Pointed to my fork so maintainers don't have to do the verifier key patch thing locally. Can revert if reviewers don't get this to work